Intermittent test failure: test_serializer_outputs_expected_schema_with_feature

Looks like an intermittent failure in main with the nimbus shared schema validation:

```
=================================== FAILURES ===================================
_ TestNimbusExperimentSerializer.test_serializer_outputs_expected_schema_with_feature _
[gw0] linux -- Python 3.9.0 /usr/local/bin/python

self = <experimenter.experiments.tests.api.v6.test_serializers.TestNimbusExperimentSerializer testMethod=test_serializer_outputs_expected_schema_with_feature>

    def test_serializer_outputs_expected_schema_with_feature(self):
        probe_set = NimbusProbeSetFactory.create()
        experiment = NimbusExperimentFactory.create_with_status(
            NimbusExperiment.Status.COMPLETE,
            firefox_min_version=NimbusExperiment.Version.FIREFOX_80,
            targeting_config_slug=NimbusExperiment.TargetingConfig.ALL_ENGLISH,
            channel=NimbusExperiment.Channel.DESKTOP_NIGHTLY,
            probe_sets=[probe_set],
        )
    
        serializer = NimbusExperimentSerializer(experiment)
        all_experiment_data = serializer.data.copy()
        arguments_data = all_experiment_data.pop("arguments")
        for experiment_data in arguments_data, all_experiment_data:
            branches_data = [dict(b) for b in experiment_data.pop("branches")]
    
>           self.assertEqual(
                experiment_data,
                {
                    "application": experiment.application,
                    "channel": experiment.channel,
                    "bucketConfig": {
                        "randomizationUnit": (
                            experiment.bucket_range.isolation_group.randomization_unit
                        ),
                        "namespace": experiment.bucket_range.isolation_group.namespace,
                        "start": experiment.bucket_range.start,
                        "count": experiment.bucket_range.count,
                        "total": experiment.bucket_range.isolation_group.total,
                    },
                    # DRF manually replaces the isoformat suffix so we have to do the same
                    "endDate": experiment.end_date.isoformat().replace("+00:00", "Z"),
                    "id": experiment.slug,
                    "isEnrollmentPaused": False,
                    "proposedDuration": experiment.proposed_duration,
                    "proposedEnrollment": experiment.proposed_enrollment,
                    "referenceBranch": experiment.reference_branch.slug,
                    "schemaVersion": settings.NIMBUS_SCHEMA_VERSION,
                    "slug": experiment.slug,
                    # DRF manually replaces the isoformat suffix so we have to do the same
                    "startDate": experiment.start_date.isoformat().replace("+00:00", "Z"),
                    "targeting": (
                        'browserSettings.update.channel == "nightly" '
                        "&& version|versionCompare('80.!') >= 0 "
                        "&& localeLanguageCode == 'en' "
                        "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
                    ),
                    "userFacingDescription": experiment.public_description,
                    "userFacingName": experiment.name,
                    "probeSets": [probe_set.slug],
                },
            )
E           AssertionError: {'schemaVersion': '1.1.0', 'slug': 'cloned-[1008 chars]ent'} != {'application': 'fenix', 'channel': 'nightl[983 chars]er']}

experimenter/experiments/tests/api/v6/test_serializers.py:37: AssertionError
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-746)
┆Issue Number: EXP-746

--- a/app/experimenter/experiments/tests/api/v6/test_serializers.py
+++ b/app/experimenter/experiments/tests/api/v6/test_serializers.py
@@ -22,6 +22,7 @@ class TestNimbusExperimentSerializer(TestCase):
         probe_set = NimbusProbeSetFactory.create()
         experiment = NimbusExperimentFactory.create_with_status(
             NimbusExperiment.Status.COMPLETE,
+            application=NimbusExperiment.Application.DESKTOP,
             firefox_min_version=NimbusExperiment.Version.FIREFOX_80,
             targeting_config_slug=NimbusExperiment.TargetingConfig.ALL_ENGLISH,
             channel=NimbusExperiment.Channel.DESKTOP_NIGHTLY,

